MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -100,6 +100,11 @@ - (NSURLRequest *)requestForEvents:(NSArray *)events {
 }
 
 - (void)setupUserAgent {
+    if ([self.applicationBundle objectForInfoDictionaryKey:@"MMEMapboxUserAgentBase"] && [self.applicationBundle objectForInfoDictionaryKey:@"MMEMapboxHostSDKVersion"]) {
```

Comment:
@rclee should the base and SDK version be split into separate conditionals? On Android I won't be overriding the SDK version so it stays as the native version in the meantime.